# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1510150034 -3600
#      Wed Nov 08 15:07:14 2017 +0100
# Node ID 547919e6aec104152c4483e4491962f15420b290
# Parent  68c4ee9545b2d52ddf2f33e5638f2f97275ff44e
colormap.m: make named colormaps case insensitive (bug #52358)

diff --git a/scripts/image/colormap.m b/scripts/image/colormap.m
--- a/scripts/image/colormap.m
+++ b/scripts/image/colormap.m
@@ -94,17 +94,17 @@ function cmap = colormap (varargin)
 
   if (! isempty (hax))
     cf = hax;
   else
     cf = get (0, "currentfigure");
   endif
 
   if (nargin == 1)
-    map = varargin{1};
+    map = tolower (varargin{1});
     if (ischar (map))
       if (strcmp (map, "default"))
         map = viridis (64);
       else
         try
           map = feval (map);
         catch
           error ("colormap: failed to set MAP <%s>", map);
